packages/core/src/components/Dropdown/__tests__/Dropdown.test.js

```diff
@@ -150,7 +150,8 @@ describe("Dropdown", () => {
 
       expect(onChange).toBeCalledTimes(2);
       expect(onChange).toHaveBeenLastCalledWith(null, {
-        action: "clear"
+        action: "clear",
+        removedValues: [mockOptions[0]]
```

Comment:
`onChange` now returns this in addition for when removing values in `isMulti` state